stdlib/public/SDK/Foundation/NSError.swift

```diff
@@ -513,7 +503,7 @@ public extension _BridgedStoredNSError
   public init(_ code: Code, userInfo: [String : Any] = [:]) {
     self.init(_nsError: NSError(domain: Self._nsErrorDomain,
                                 code: numericCast(code.rawValue),
-                                userInfo: _stringDictToAnyHashableDict(userInfo)))
+                                userInfo: userInfo))
```

Comment:
So I speculatively tried putting a `fatalError()` here and none of the tests failed 👀

Would you mind adding some additional coverage for this case?

The one above did fairly bigly when I did the same, so it's just this one.

Reply:
There are currently no objects conforming to `_BridgeStoredNSError` with a `Code.RawValue` that is an `UnsignedInteger`. Is this default implementation still needed?